### PR TITLE
enhance multikey validation error message

### PR DESCRIPTION
**Pull request by gupadhyaya**
fixes https://github.com/harmony-one/harmony/issues/2696

### CODE REVIEW

**Comment by gupadhyaya** [comment]
output:
```
cannot set up node account: bls keys do not belong to same shard
key: 2258b3b7b2b95d528467ddbfd25d8b551f1917d0589ee64dc13b90c3364a62627f3ec139a7f76fb82d0832cb404a478c, shard id: 0
key: c167fc725b898faabf54fc30ad073419f38b94a4cadeced6957a65cc3810fa631003b19f165101b61f0e2bfa027c6e11, shard id: 1
key: cc7d8215460868af2731d01a0ecf0a3159c3f0117bc26f7fd977c92542c725d6c1201a8031ef6581e9bf1e2564ffa091, shard id: 1
```

File: internal/configs/node/config.go [hunk 1]
```diff
@@ -251,37 +252,45 @@ func SetShardingSchedule(schedule shardingconfig.Schedule) {
...
 	epoch := conf.networkType.ChainConfig().StakingEpoch
 	numShards := conf.shardingSchedule.InstanceForEpoch(epoch).NumShards()
 	shardID := new(big.Int).Mod(pubKey.Big(), big.NewInt(int64(numShards)))
 	return uint32(shardID.Uint64()), nil
 }
 
+// ShardIDFromConsensusKey returns the shard ID statically determined from the
+// consensus key.
+func (conf *ConfigType) ShardIDFromConsensusKey() (uint32, error) {
+	return conf.ShardIDFromKey(conf.ConsensusPubKey.PublicKey[0])
```
Comment (fxfactorial) [inline internal/configs/node/config.go:272]: this [0] will never explode?

File: internal/configs/node/config.go [hunk 2]
```diff
@@ -251,37 +252,45 @@ func SetShardingSchedule(schedule shardingconfig.Schedule) {
...
+		if shardID != sID {
+			isSameShard = false
 		}
+		keyShardStrs = append(
+			keyShardStrs,
+			fmt.Sprintf("key: %s, shard id: %d", key.SerializeToHexStr(), shardID),
+		)
+	}
+	if !isSameShard {
+		return fmt.Errorf("bls keys do not belong to same shard\n%s", strings.Join(keyShardStrs, "\n"))
```
Comment (fxfactorial) [inline internal/configs/node/config.go:293]: lets get rid of all `fmt.Errof` and only use `pkg/error`, therefore `errors.Errof`

File: internal/configs/node/config.go (hunk 1 above)
Comment (gupadhyaya) [reply internal/configs/node/config.go:272]: nope. at least one bls key is always provided, including explorer for which default dummy bls is used.

**Pull request merged**